### PR TITLE
Added value for external_db

### PR DESCRIPTION
**Pull request by geoffrey1330**
Added feature to support external db

### CODE REVIEW

File: charts/ever-gauzy-api/templates/deployment.yaml [hunk 1]
```diff
@@ -60,7 +60,11 @@ spec:
             - name: SENTRY_POSTGRES_TRACKING_ENABLED
               value: {{ .Values.sentry.sentryPostgresTrackingEnabled | quote }}
             - name: DB_HOST
+            {{- if .Values.postgresql.global.external_db }}
+              value: {{ .Values.postgresql.global.postgresql.auth.username | quote }}
```
Comment (evereq) [inline charts/ever-gauzy-api/templates/deployment.yaml:64]: for DB_HOST you want to use `username`??

Comment (geoffrey1330) [reply charts/ever-gauzy-api/templates/deployment.yaml:64]: Nah it a typo didn't commit my last change.

Comment (geoffrey1330) [reply charts/ever-gauzy-api/templates/deployment.yaml]: committed

Comment (evereq) [reply charts/ever-gauzy-api/templates/deployment.yaml]: OK, I will merge, but what about DB_CERT? Not needed in AWS RDS or?

**Pull request merged**